core/lib/types/src/event/mod.rs

```diff
@@ -42,14 +43,21 @@ impl VmEvent {
     }
 }
 
-impl From<&VmEvent> for ApiVmEvent {
+impl From<&VmEvent> for Log {
     fn from(vm_event: &VmEvent) -> Self {
-        ApiVmEvent {
-            l1_batch_number: vm_event.location.0,
-            tx_batch_index: vm_event.location.1,
+        Log {
             address: vm_event.address,
-            indexed_topics: vm_event.indexed_topics.clone(),
-            value: hex::encode(&vm_event.value),
+            topics: vm_event.indexed_topics.clone(),
+            data: Bytes::from(vm_event.value.clone()),
+            block_hash: None,
+            block_number: None,
+            l1_batch_number: Some(U64::from(vm_event.location.0 .0)),
+            transaction_hash: None,
```

Comment:
nit: do we have any meaningful `transaction_hash` that we can put here? In other words, is there a transaction hash already computed by this point? If not, feel free to keep as is